src/test/java/posters/pageobjects/components/MiniCart.java

```diff
@@ -38,22 +38,22 @@ public class MiniCart extends AbstractComponent
     @Step("ensure availability mini cart")
     public void isComponentAvailable()
     {
-        $("#btnCartOverviewForm").should(exist);
+        $("#header-customer-menus").should(exist);
     }
 
     /// ----- mini cart navigation ------ ///
 
     @Step("open the mini cart")
     public void openMiniCart()
     {
-        headerCart.hover();
-        miniCart.waitUntil(visible, 9000);
+        headerCart.click();
+        //miniCart.waitUntil(visible, 9000);
```

Comment:
first: there should be a validation if the mini cart is open
second: please don't keep out commented lines, if they are not needed